host/CMakeLists.txt

```diff
@@ -253,6 +283,7 @@ endif ()
 
 # For including edge routines.
 target_include_directories(oehost PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
+target_include_directories(oehostverify PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
This shouldn't be necessary since there's no dependency on oeedger8r generated headers, no?

Reply:
Good point. It shouldn't be necessary.